include/util/vector_view.hpp

```diff
@@ -143,8 +143,33 @@ template <> class vector_view<bool>
     unsigned *m_ptr;
     std::size_t m_size;
 
+    static constexpr std::size_t UNSIGNED_BITS = CHAR_BIT * sizeof(unsigned);
+
   public:
     using value_type = bool;
+    struct reference
```

Comment:
This needs to be lower-case to be compatible with the vector naming of this type.